content/docs/gke/authentication.md

```diff
@@ -7,6 +7,39 @@ weight = 4
 
 ## In-cluster authentication
 
+Starting from Kubeflow version 0.6, you can consume Kubeflow from custom namespaces (i.e., namespaces other than `kubeflow`).
+The `kubeflow` namespace is intended for running Kubeflow system components while individual jobs and model deployments
+are expected to run in separate namespaces. In order to do this you will need to install GCP credentials into the new namespace.
+
+### Starting in 0.7: GKE Workload Identity
+
+Starting in 0.7, we use the new GKE feature: [workload identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity).
+This is the recommended way to access GCP APIs from your GKE cluster.
+We will no longer download GCP service account key. Instead, we configure a Kubernetes service account (KSA) to act as a GCP service account (GSA).
+
+In your user namespace (e.g. kubeflow-NAME), there should be a KSA called default-editor. This KSA should have an annotation of the corresponding GSA:
+
+```
+kubectl -n kubeflow-NAME describe serviceaccount default-editor
```

Comment:
nit here and below in commands use bash variables to indicate variables
e..g

```
KFNAMESPACE=<your kubeflow namespace>
kubectl -n ${KFNAMESPACE} ...
```

Reply:
done